src/app/components/login/login.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core';
+import { AngularFireAuth } from '@angular/fire/auth';
+import { auth as Auth, User } from 'firebase/app';
+import * as FullStory from '@fullstory/browser';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent {
+
+  user: User | null;
+
+  constructor(private afa: AngularFireAuth) {
+    afa.user.subscribe(user => {
```

Comment:
nit: idunno if `afa` is an AngularFire convention, but `auth` is a more interpretable name